Shaders/PostProcess/ScreenSpaceAmbientOcclusion/private/SSAO_ComputePrefilteredDepthBuffer.fx

```diff
@@ -4,7 +4,9 @@
 #include "FullScreenTriangleVSOutput.fxh"
 #include "PostFX_Common.fxh"
 
-#pragma warning(disable : 3078)
+#if !defined(GLSL) && !defined(GL_ES)
+    #pragma warning(disable : 3078)
```

Comment:
Why is this pragma needed BTW?

Reply:
In FXC compiler `for` loop  doesn't work as in C++, iterated variable in global visibility scope